packages/boot/README.md

```diff
@@ -0,0 +1,84 @@
+# @loopback/boot
+
+A collection of Booters for LoopBack Applications
+
+# Overview
+
+A Booter is a Class that can be bound to an Application and is called
+to perform a task before the Application is started. A Booter may have multiple
+phases to complete it's task.
+
+An example task of a Booter may be to discover and bind all artifacts of a
+given type.
+
+## Installation
+
+```shell
+$ npm i @loopback/boot
+```
+
+## Basic Use
+
+```ts
+import {ControllerBooter} from '@loopback/boot';
+app.booter(ControllerBooter); // register booter
```

Comment:
Interesting. How are we going to register all booters, will we need another metabooter?

Reply:
I'm not sure I follow. A booter is registered by `app.booter`. A normal Application developer will likely not be writing their own Booters. We can/should add support for a `ScriptBooter` however at some point down the road to boot all scripts in a scripts folder similar to how it is done today.